packages/web-app-files/src/views/Personal.vue

```diff
@@ -198,16 +204,73 @@ export default {
   },
 
   methods: {
-    ...mapActions('Files', ['loadIndicators', 'loadPreview']),
+    ...mapActions('Files', ['loadIndicators', 'loadPreview'], 'showMessage'),
     ...mapMutations('Files', [
       'SELECT_RESOURCES',
       'SET_CURRENT_FOLDER',
       'LOAD_FILES',
       'CLEAR_CURRENT_FILES_LIST',
-      'UPDATE_CURRENT_PAGE'
+      'REMOVE_FILE_FROM_SEARCHED'
     ]),
     ...mapMutations(['SET_QUOTA']),
 
+    async fileDropped(fileIdTarget) {
+      const selected = this.selectedFiles
+      const targetInfo = this.activeFiles.find(e => e.id === fileIdTarget)
+      const targetSelected = selected.some(e => e.id === fileIdTarget)
+      if (targetSelected) return
+      if (targetInfo.type !== 'folder') return
+
+      this.loading = true
+      const targetPath = targetInfo.path + '/'
+      const errors = []
+      const itemsInTarget = await this.getFolderItems(targetPath)
+      for (let i = 0; i < selected.length; i++) {
+        const current = selected[i]
+        const exists = itemsInTarget.some(e => basename(e.name) === current.name)
+        if (exists) {
+          const message = this.$gettext('Resource with name %{name} already exists')
+          errors.push({
+            resource: current.name,
+            message: this.$gettextInterpolate(message, { name: current.name }, true)
+          })
+          continue
+        }
+        try {
+          await this.$client.files.move(current.path, targetPath + current.name)
```

Comment:
Add before this line:
`const targetPath = join(targetInfo.path, current.name)`
and import `join` from `path`.
Then you can use `targetPath` instead of the string concatenation within the `move` call.